community/values/src/main/java/org/neo4j/values/storable/CRSTable.java

```diff
@@ -17,36 +17,43 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.values.virtual;
+package org.neo4j.values.storable;
 
-public enum CoordinateReferenceSystem
+public enum CRSTable
 {
-    Cartesian( "cartesian", 7203, "http://spatialreference.org/ref/sr-org/7203/" ),
-    WGS84( "WGS-84", 4326, "http://spatialreference.org/ref/epsg/4326/" );
-
+    CUSTOM( "custom", 0 ),
+    EPSG( "epsg", 1 ),
+    SR_ORG( "sr-org", 2 );
     public final String name;
```

Comment:
same here as before why those field a republic?

Reply:
I always use public fields for enums because they are immutable. Seems a common pattern, but we can do it differently with accessor methods if that is preferred for stylistic reasons.

Reply:
yeah i would prefer access methods